Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Copyright Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using UnityEngine;
+
+/// <summary>
+/// Class for manually controlling the camera when HuP isn't running. Attach to same CameraRig object as the StereoCamera.cs script.
+/// </summary>
+public class ManualCameraControl : MonoBehaviour
+{
+    public enum MouseButton
+    {
+        Left,
+        Right,
+        Middle,
+        Control,
+        Shift,
+        Focused,
+        None
+    }
+
+    public float ExtraMouseSensitivityScale = 3.0f;
+    public float DefaultMouseSensitivity = 0.1f;
+    public MouseButton MouseLookButton = MouseButton.Control;
+    public bool IsControllerLookInverted = true;
+    private bool isMouseJumping = false;
+
+    private bool isGamepadLookEnabled = true;
+    private bool isFlyKeypressEnabled = true;
+    private Vector3 lastMousePosition = Vector3.zero;
+    private Vector3 lastTrackerToUnityTranslation = Vector3.zero;
+    private Quaternion lastTrackerToUnityRotation = Quaternion.identity;
+    private bool appHasFocus = true;
+    private bool wasLooking = false;
+
+    private static float InputCurve(float x)
+    {
+        // smoothing input curve, converts from [-1,1] to [-2,2]
+        return Mathf.Sign(x) * (1.0f - Mathf.Cos(0.5f * Mathf.PI * Mathf.Clamp(x, -1.0f, 1.0f)));
+    }
+
+#if UNITY_EDITOR
```

Comment:
Perhaps refactor this into a new class in the Utilities folder called DLLImports?